crypto/x509/x509_lu.c

```diff
@@ -370,7 +370,7 @@ int X509_OBJECT_up_ref_count(X509_OBJECT *a)
     return 1;
 }
 
-X509 *X509_OBJECT_get0_X509(X509_OBJECT *a)
+X509 *X509_OBJECT_get0_X509(const X509_OBJECT *a)
```

Comment:
I have found any doc about X509_OBJECT type and his related methods.
It's may be considered as an internal data type ?